include/tvm/runtime/device_api.h

```diff
@@ -36,6 +36,39 @@ constexpr int kTempAllocaAlignment = 64;
 /*! \brief Maximum size that can be allocated on stack */
 constexpr int kMaxStackAlloca = 1024;
 
+/*! \brief The default device allocated to an operator */
+constexpr DLDeviceType kDLDefaultDevice = kDLCPU;
```

Comment:
by this `default` , do you mean the fallback device?

Reply:
@jackwish Yes.

Reply:
this should not appear here, but instead we should pass fallback device as argument or put it in a context

Reply:
@tqchen +1